### PR TITLE
Fix/ui etl fancy edit properties

### PR DESCRIPTION
**Pull request by ajainarayanan**
Opened it against release/3.0 (Reference https://github.com/caskdata/cdap/pull/2319)

- [ ] Add same view to Sink property edit (https://github.com/caskdata/cdap/pull/2329)
- [ ] Add a sample JSON file for sink to be used in UI. (https://github.com/caskdata/cdap/pull/2329)

### CODE REVIEW

File: cdap-ui/app/directives/widget-container/widget-container.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+angular.module(PKG.name + '.commons')
+  .directive('widgetContainer', function($compile) {
+    var registry = {
```
Comment (nitinmotgi) [inline cdap-ui/app/directives/widget-container/widget-container.js]: Remove if un-used.

File: cdap-ui/app/directives/widget-container/widget-container.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            angularElement.attr('placeholder', scope.myconfig[scope.name].info);
+            element.append(angularElement);
+            break;
+          case 'password':
+            angularElement = angular.element('<input class="form-control" ng-model="model"/>');
+            angularElement.attr('type', 'password');
+            angularElement.attr('placeholder', scope.myconfig[scope.name].info);
+            element.append(angularElement);
+            break;
+          case 'textarea':
```
Comment (nitinmotgi) [inline cdap-ui/app/directives/widget-container/widget-container.js]: We don't have textarea, so don't know what is this meant to do.

File: cdap-ui/app/features/etlapps/templates/create/tabs/sourcePropertyEdit.html [hunk 3]
```diff
@@ -1,28 +1,31 @@
-<div class="container">
-  <div class="row" ng-init="isSaved=false">
-    <div class="col-xs-12">
-      <h3>Editing {{source.name}} Properties</h3>
-      <div class="text-success" ng-if="isSaved">
-        Saved!
-      </div>
-      <form class="form-horizontal">
+<h3>Editing {{source.name}} Properties</h3>
```
Comment (nitinmotgi) [inline cdap-ui/app/features/etlapps/templates/create/tabs/sourcePropertyEdit.html]: ```{{source.name}} properties``` remove Editing

File: cdap-ui/templates/etlRealtime/JmsSource.json [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
-             },
+  "id": "JmsSource",
+  "groups" : {
+    "position": [ "group1" ],
+    "group1": {
+       "display" : "JMS Configuration",
+       "position" : [ "jms.destination.name", "jms.messages.receive" ],
+       "fields" : {
+          "jms.destination.name" : {
+             "widget-type": "textbox",
```
Comment (nitinmotgi) [inline cdap-ui/templates/etlRealtime/JmsSource.json]: - [] Changed widget-type to widget
- [] Replace widget- with <empty>

**Comment by ajainarayanan** [comment]
This PR is dependent on this PR (https://github.com/caskdata/cdap/pull/2329) to be merged to release/3.0

**Comment by elfenheart** [comment]
:+1: LGTM

**Comment by nitinmotgi** [comment]
lgtm :+1:

**Pull request merged**